packages/fxa-settings/src/pages/Index/index.tsx

```diff
@@ -16,19 +16,100 @@ import {
   isClientPocket,
   isClientRelay,
 } from '../../models/integrations/client-matching';
-import { isOAuthIntegration } from '../../models';
+import { isOAuthIntegration, useFtlMsgResolver } from '../../models';
+import GleanMetrics from '../../lib/glean';
+import { getLocalizedErrorMessage, interpolate } from '../../lib/error-utils';
+import Banner from '../../components/Banner';
+import { isEmailMask, isEmailValid } from 'fxa-shared/email/helpers';
+import { AuthUiErrors } from '../../lib/auth-errors/auth-errors';
 
 export const Index = ({
   integration,
   serviceName,
-}: IndexProps & RouteComponentProps) => {
+  signUpOrSignInHandler,
+  prefillEmail,
+  deleteAccountSuccess,
+  hasBounced,
+}: IndexProps) => {
   const clientId = integration.getClientId();
   const isSync = integration.isSync();
   const isDesktopRelay = integration.isDesktopRelay();
   const isOAuth = isOAuthIntegration(integration);
   const isPocketClient = isOAuth && isClientPocket(clientId);
   const isMonitorClient = isOAuth && isClientMonitor(clientId);
   const isRelayClient = isOAuth && isClientRelay(clientId);
+
+  const ftlMsgResolver = useFtlMsgResolver();
+  const [errorBannerMessage, setErrorBannerMessage] = useState('');
+  const [successBannerMessage, setSuccessBannerMessage] = useState(
+    deleteAccountSuccess
+      ? ftlMsgResolver.getMsg(
+          'index-account-delete-success',
+          'Account deleted successfully'
+        )
+      : undefined
+  );
+  const [tooltipErrorText, setTooltipErrorText] = useState(
+    hasBounced
+      ? ftlMsgResolver.getMsg(
+          'auth-errors-1018',
+          'Your confirmation email was just returned. Mistyped email?'
+        )
+      : undefined
+  );
+
+  useEffect(() => {
+    // Note we might not need this due to automatic page load events,
+    // but it's here for now to match parity
+    GleanMetrics.emailFirst.view();
+  }, []);
+
+  const { handleSubmit, register } = useForm<IndexFormData>({
+    mode: 'onChange',
+    criteriaMode: 'all',
+    defaultValues: {
+      email: '',
+    },
+  });
+
+  const onSubmit = async ({ email }: IndexFormData) => {
+    if (!isEmailValid(email)) {
+      setTooltipErrorText(
+        ftlMsgResolver.getMsg('auth-error-1011', 'Valid email required')
+      );
+      return;
+    } else if (isEmailMask(email)) {
```

Comment:
Just realized we should only do some of these checks only if the account doesn't exist. Going to make a couple tweaks around this section now.